Dockerfile

```diff
@@ -1,53 +1,63 @@
-FROM debian:jessie
+FROM erlang:20.3 AS build-env
 
-MAINTAINER Octavo Labs AG info@vernemq.com
+WORKDIR /tmp/vernemq-build
 
-RUN apt-get update && apt-get install -y \
-    libssl-dev \
-    logrotate \
-    sudo \
-    curl \
-    jq \
-    && rm -rf /var/lib/apt/lists/*
-
-ENV VERNEMQ_VERSION 1.6.1
+ARG VERNEMQ_VERSION=1.6.0
+ARG TARGET rel
+ARG VERNEMQ_REPO=https://github.com/vernemq/vernemq.git
 
 # Defaults
 ENV DOCKER_VERNEMQ_KUBERNETES_APP_LABEL vernemq
 ENV DOCKER_VERNEMQ_LOG__CONSOLE console
 
-ADD https://bintray.com/artifact/download/erlio/vernemq/deb/jessie/vernemq_$VERNEMQ_VERSION-1_amd64.deb /tmp/vernemq.deb
+RUN \
+    apt-get update \
+    && apt-get -y install build-essential git libssl-dev  \
+	&& rm -rf /var/lib/apt/lists/* \
+    && git clone -b $VERNEMQ_VERSION --single-branch --depth 1 $VERNEMQ_REPO .
 
-RUN dpkg -i /tmp/vernemq.deb
-RUN rm /tmp/vernemq.deb
+ADD bin/build.sh build.sh
 
-ADD files/vm.args /etc/vernemq/vm.args
-ADD bin/vernemq.sh /usr/sbin/start_vernemq
-ADD bin/rand_cluster_node.escript /var/lib/vernemq/rand_cluster_node.escript
+RUN ./build.sh $TARGET
 
 
-# MQTT
-EXPOSE 1883
+FROM debian:stretch
```

Comment:
Have you considered the -slim variant of stretch? It should really help in keeping the image size reasonable.